.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: monthly
+    ignore:
+      - dependency-name: the-last-browser-launcher
```

Comment:
Note, this was added to the ignore list automatically because the version is pinned in `package.json`:

https://github.com/airtap/system/blob/962594f14533d824d621f8a7eb5d4127eb07fa70/package.json#L22